src/components/charts/BarChart.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="bar-chart-container" v-if="loaded">
+    <Bar
+      id="my-chart-id"
+      :options="chartOptions"
+      :data="chartData"
+    />
+  </div>
+  <div v-else>
```

Comment:
issue: Use either of these instead
* https://vue.carbondesignsystem.com/vue2/?path=/story/components-cvinlineloading--default
* https://vue.carbondesignsystem.com/vue2/?path=/story/components-cvloading--default